app/views/galleries/index.rss.builder

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+self.class.send(:include, GalleryDisplayingView)
+
+def gallery_image(gallery)
+  unless gallery.images.count.zero?
+    presenter = presenter_for_gallery_image(gallery.images.first)
+    image_url = presenter.field_value('edmPreview')
```

Comment:
We should not be using edmPreview as-is, but converting it to the thumbnail API equivalent. Just reuse `GalleryDisplayingView#gallery_image_thumbnail` which does just that.